crawler/db.py

```diff
@@ -259,3 +268,72 @@ def init_warehouse_db_command():
     mysql_conn.close()
 
     logger.debug("Done")
+
+
+def create_dart_sql_server_conn(config: ModuleType, readonly=True) -> pyodbc.Connection:
+    """Create a SQL Server connection to DART with the given config parameters.
+
+    Arguments:
+        config {ModuleType} -- application config specifying database details
+
+    Returns:
+        pyodbc.Connection -- connection object used to interact with the sql server database
+    """
+    dart_db_host = config.DART_DB_HOST  # type: ignore
+    dart_db_port = config.DART_DB_PORT  # type: ignore
+    if readonly:
+        dart_db_username = config.DART_DB_RO_USER  # type: ignore
+        dart_db_password = config.DART_DB_RO_PASSWORD  # type: ignore
+    else:
+        dart_db_username = config.DART_DB_RW_USER  # type: ignore
+        dart_db_password = config.DART_DB_RW_PASSWORD  # type: ignore
+    dart_db_db = config.DART_DB_DBNAME  # type: ignore
+    dart_db_driver = config.DART_DB_DRIVER  # type: ignore
+
+    connection_string = 'DRIVER='+dart_db_driver+';SERVER='+dart_db_host+f';PORT={dart_db_port};DATABASE='+dart_db_db+';UID='+dart_db_username+';PWD=' + dart_db_password
```

Comment:
```suggestion
    connection_string = f'DRIVER={dart_db_driver};SERVER={dart_db_host};PORT={dart_db_port};DATABASE={dart_db_db};UID={dart_db_username};PWD={dart_db_password}'
```

Reply:
I've made (and tested) these changes and will push up once I've addressed all comments